site/docs/latest/functions/quickstart.md

```diff
@@ -212,9 +212,9 @@ $ touch reverse.py
 In that file, add the following:
 
 ```python
-from pulsarfunction import pulsar_function
+from pulsar import Function
 
-class Reverse(pulsar_function.PulsarFunction):
+class Reverse(Function):
```

Comment:
`class Reverse:` 
or 
`class Reverse(Object):`

Reply:
Ah, you're right. It won't be needed in this case.